.github/workflows/rebuild-all.yaml

```diff
@@ -14,5 +14,5 @@ jobs:
       - name: Repository Dispatch
         uses: peter-evans/repository-dispatch@v2
         with:
-          token: ${{ secrets.PAT_GITHUB_DISPATCH }}
+          token: ${{ secrets.GH_TOKEN_NEXTSTRAIN_BOT_WORKFLOW_DISPATCH }}
```

Comment:
I didn't realize we were using a third party action to create a repo dispatch until now. Should we be worried about passing org secrets to a third party action here?

Reply:
Updated in f33b6d1836471dcd531d735ba0d9e8c7566d3d0a to stop using a third-party action.